lib/device_api/android.rb

```diff
@@ -38,6 +39,8 @@ def self.get_device_type(options)
       case Device.new(serial: options.keys.first).manufacturer.downcase
         when 'amazon'
           type = :kindle
+        when 'samsung'
```

Comment:
Indent `when` as deep as `case`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.